OpenRA.Mods.Common/Traits/Buildings/BridgePlaceholder.cs

```diff
@@ -0,0 +1,80 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Linq;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Placeholder actor that transforms into another actor type when repaired.")]
+	class BridgePlaceholderInfo : ITraitInfo
+	{
+		public readonly string Type = "GroundLevelBridge";
+
+		[FieldLoader.Require]
+		[Desc("Actor type to replace with on repair.")]
+		[ActorReference] public readonly string ReplaceWithActor;
```

Comment:
MSBuild complains with a warning that this is never assigned anything and always remains `null`. Not a big deal and I guess the next PR will adress that, but I wanted to point it out just in case.